src/modules/fancyzones/FancyZonesLib/WorkArea.cpp

```diff
@@ -57,13 +57,13 @@ namespace
 
     public:
 
-        HWND NewZoneWindow(Rect position, HINSTANCE hinstance, WorkArea* owner)
+        HWND NewWindow(Rect position, HINSTANCE hinstance, WorkArea* owner)
```

Comment:
Here the _Zone_ matters. The reason for it is that becomes unclear which _NewWindow_ do you mean. WorkArea window? Zone window? Any other window snapped to a zone?
So let's keep it _NewZoneWindow_

Reply:
Would you like it better if I renamed the class to `WorkAreaWindowPool`?

Or rename the method to `NewWorkAreaWindow`?

Or is the previous name clear enough for now?

Reply:
My main goal in creating this PR was to remove the use of the term "zone window". So any renaming you can think of that would make this term gone is a success IMO.

Reply:
In case if you name `ZoneWindow` -> `ZoneOverlay` so there should be `NewZoneOverlayWindow` isn't it?

Reply:
Renamed to `NewZonesOverlayWindow`.